openeo/Load_Collection.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to load a data collection?\n",
+    "\n",
+    "This notebook provides a detailed guide on how to load a data collection, including all the necessary steps involved. Additionally, it will explain how to authenticate your account to ensure secure access to the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import packages\n",
+    "import openeo\n",
+    "\n",
+    "# connect to the backend\n",
+    "connection = openeo.connect(url = \"openeo-staging.creo.vito.be\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To verify whether they have been authenticated to load collections, users can check their connection to the backend. If the connection is established, they can be confident that they are authenticated and can proceed with loading the collections."
```

Comment:
> If the connection is established, they can be confident that they are authenticated and can proceed with loading the collections.

This is misleading: a connection can be "established" also without authentication

Reply:
updated